userdocs/profiles.dev/docs/author-docs/nested-profiles.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 5
+---
+
+# Nested profile artifacts
+
+Profiles can refer to other profiles in their list of artifacts.
+
+Simply use the `source` field:
+
+```yaml
+# ...
+spec:
+  # ...
+  artifacts:
+    - name: # the name of your artifact as you would like it to be known in the profile
+      source:
+        url: # required: fully qualified URL to the nested profile repository
+        branch: # optional: the repo branch if the profile is somewhere other than `main`
+        path: # optional: the relative path to a profile's directory within the repo
+        tag: # optional: the tag of the profile
+	# ...
+```
+
```

Comment:
shall we add an example?  Also the yaml is wrong:
```
    - name: nested-profile
      profile:
        source:
          url: https://github.com/weaveworks/profiles-examples
          tag: bitnami-nginx/v0.0.1
```

Reply:
thanks, missed the `profile` key somehow 🤷‍♀️